File: test-workspaces/modules/shared_common/src/taxonomy_aes205_cycle_start.py
# AES205: Part of circular dependency chain
from ..taxonomy.taxonomy_aes205_cycle_end import CycleEndEntity

class CycleStartEntity:
    def __init__(self):
        self.data = "start"

    def process(self):
        end = CycleEndEntity()
        return self.data

File: workspaces-bad/packages/shared_common/src/taxonomy_aes205_cycle_start.ts
// AES205: Part of circular dependency chain
import { CycleEndEntity } from './taxonomy_aes205_cycle_end';

export class CycleStartEntity {
    data: string = "start";
    process(): string {
        const end = new CycleEndEntity();
        return this.data;
    }
}
